src/components/core/TextButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { merge } from 'lodash'
+import { ButtonProps, Pressable, PressableProps, Text } from 'react-native'
+
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(({ typography, palette }, { variant }) => {
+  const variantStyles = {
+    primary: {
+      text: {
+        color: palette.primary
+      }
+    },
+    secondary: {
+      text: {
+        color: palette.secondary
+      }
+    }
+  }
+
+  const baseStyles = {
+    text: {
+      ...typography.body
+    }
+  }
+
+  return merge(baseStyles, variantStyles[variant])
+})
+
+type TextButtonProps = PressableProps &
+  ButtonProps & {
+    variant: 'primary' | 'secondary'
+  }
+
+export const TextButton = (props: TextButtonProps) => {
+  const { title, variant, ...other } = props
```

Comment:
more of a comment on ReactNative, but why did they choose to call the text content `title`? It breaks my web dev brain :(

Reply:
no clue, i would have used text, or just children itself for buttons, but react-native decided, and id rather keep the prop convention the same.....

Reply:
should keep it the same yes 💯  just complaining haha

Reply:
totally